doc/sphinx/source/changelog.rst

```diff
@@ -3,6 +3,129 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+- Two new recipes have been added:
+
+  - Recipe :ref:`recipe_aod_aeronet_assess.yml <recipe_aod_aeronet_assess>`
+    evaluates model aerosol optical depth (AOD) climatologies against ground
+    based observations from the AeroNET measurement network.
+  - Recipe :ref:`recipe_climate_patterns.yml <recipe_climate_patterns>`
```

Comment:
```suggestion
  - Recipe :ref:`recipe_climate_patterns.yml <recipes_climate_patterns>`
```
Looks like the label uses an extra `s`: https://github.com/ESMValGroup/ESMValTool/blob/bc25f8723634a8f57f43fd89646c8da1f78ba8cd/doc/sphinx/source/recipes/recipe_climate_patterns.rst?plain=1#L1

Reply:
Argh! Apologies! Fixed in ae850fe5b 😊